TACOS-service/src/main/java/cz/fi/muni/TACOS/service/UserService.java

```diff
@@ -44,4 +44,19 @@
      */
     void removeOrderFromSubmittedOrders(User user, Order order);
 
+    /**
+     * Create user
+     *
+     * @param user which will be created
+     */
+    void createUser(User user);
```

Comment:
Why did u created new create method? There should already be one declared in EntityService interface.